CorsixTH/Lua/world.lua

```diff
@@ -905,13 +836,13 @@ function World:previousSpeed()
   end
 end
 
--- Return if the selected speed the same as the current speed.
+--! Return true if the given speed the same as the current speed.
```

Comment:
It's `!return` for luadoc isn't it?

Reply:
Yes, but convention is to prefer the method summary to the return parameter description when doing both would be redundant. The change is good.

Reply:
@lewri: I just aimed at getting the line recognized in luadoc, as it seemed mostly complete although I agree not entirely according to luadoc.